test/index.ts

```diff
@@ -1,15 +1,20 @@
 import * as request from 'supertest';
 import { startServer } from '../src/setup';
 import { expect } from 'chai';
+import * as dotenv from 'dotenv';
+import { User } from '../src/entity/user';
+import * as bcrypt from 'bcrypt';
 
 before(async () => {
+  dotenv.config({ path: __dirname + '/../test.env' });
   await startServer();
+  const userRepository = User.getRepository();
+  await userRepository.clear();
 });
 
-describe('Print test', () => {
-  it('Printing', () => {
-    console.log('Test ran');
-  });
+after(async () => {
+  const userRepository = User.getRepository();
+  await userRepository.clear();
```

Comment:
ah, agora que eu reparei nesse after *fora* do describe do createUser...

dentro de cada describe vc pode incluri tb os:
- before (roda no começo do describe)
- beforeEach (roda antes de cada it)
- afterEach (roda depois de cada it)
- after (roda depois de executar todos os it)

Conforme o Alan comentou na call de ambientes, é interessante que a cada teste (i.e., a cada `it`) o banco deve ser limpado para não afetar os testes. Com isso, sugiro que além desse `after`, tenha um afterEach ou beforeEach dentro do describe `createUser` para garantir que cada it vai trabalhar com o banco vazio, sem interferencias de outros testes.

Reply:
Eu não tinha adicionado `afterEach` pois eu estava utilizando o mesmo usuário criado no teste da mutation pra poder testar o hash da senha. Agora alterei os testes pra criar o usuário de novo em todo `it` e adicionei o `afterEach`